src/sandstorm/sandstorm-http-bridge-internal.capnp

```diff
@@ -30,6 +30,9 @@ struct BridgeObjectId {
   # Recall that Sandstorm obfuscates object IDs automatically, such that clients cannot see the
   # contents and the app can trust that the ID passed from Sandstorm is authentic. Hence, we can
   # put all the metadata we need directly in this structure and let Sandstorm store it.
+  #
+  # Note the AppHooks.BridgeObjectId defined in `sandstorm-http-bridge.capnp` is wire
```

Comment:
Instead of defining parallel, compatible messages, can you instead pull out the `application` part of this message and pass only that on to the app? I think that's what I had in mind here.

Reply:
I agree that would be a better API, but it seems much more complex to implement -- unless I'm missing something, the bridge would have to intercept all incoming calls from the platform to AppPersistent.save(), so that it could wrap the sturdyref. I'm not sure how to approach that in way that seems worth the trouble. Open to ideas.

Reply:
This is what membranes are for. :) Take a look at how the supervisor process uses membranes to intercept `save()` calls destined for the app and then wraps them into [SupervisorObjectId](https://github.com/sandstorm-io/sandstorm/blob/master/src/sandstorm/supervisor.capnp#L381).

https://github.com/sandstorm-io/sandstorm/blob/master/src/sandstorm/supervisor.c++#L1716

Reply:
Aha, I didn't realize the capnproto supported this in a generic way -- that makes this much easier. I'll update the schema.

Reply:
Updated.